src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -61,6 +64,88 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
         };
     }
 
+    @computed get mutationRateSummary() {
+        if (this.props.store.gene.result &&
+            this.props.store.patientIds.result &&
+            this.props.store.mutationData.result &&
+            this.props.store.mskImpactGermlineConsentedPatientIds.result) {
+
+            const germlineMutationRate = this.germlineMutationRate(this.props.store.mutationData.result,
+                                                                   this.props.store.patientIds.result,
+                                                                   this.props.store.mskImpactGermlineConsentedPatientIds.result)
+            const germlineMutationRateElement:JSX.Element|null =
+                (this.props.store.mskImpactGermlineConsentedPatientIds.result.length
```

Comment:
there's a weird relationship here.  how can germlineMutationRate be above zero if there are no germline consented patients.  i guess same question as as above.  probably just my ignorance.